### PR TITLE
1st commit

### PR DESCRIPTION
**Pull request by hyunyong**
I'm stuck with at the below error.

undefined reference to `std::type_info const& edm::typelookup::classTypeInfo<QC8::MuonDDDConstants>()'

it took already a long time, so I made pr.
I am still working for debugging.

### CODE REVIEW

**Comment by jshlee** [comment]
ok remove root file and i will have a look

File: Analysis/data/gem11S_c2_r1.xml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>
```
Comment (jshlee) [inline Analysis/data/gem11S_c2_r1.xml:1]: why is this not in geometry?>

**Comment by hyunyong** [comment]
Giovanni added, he made script for building geometry.

File: Analysis/python/excel_to_csv.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import xlrd, openpyxl
```
Comment (jshlee) [inline Analysis/python/excel_to_csv.py:1]: these would be better in scripts if they to used from command line

**Comment by jshlee** [comment]
add readme to install

File: Geometry/python/__init__.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#Automatically created by SCRAM
```
Comment (jshlee) [inline Geometry/python/__init__.py:1]: these should be added to gitignore

**Pull request closed without merging**